docs/developer_guide/env_setup.md

```diff
@@ -109,6 +109,29 @@ Without setting the enviromnent minikube might not find the docker images when d
 1. run the image build from the yunikorn-k8shim repository root: `make image`
 1. deploy the scheduler as per the normal instructions.
 
+## Local Kubernetes Cluster with Kind
+
+Kind (Kubernetes in Docker) is a lightweight tool for running lightweight Kubernetes environments.  It is very easy to test different Kubernetes versions with kind.  You can just select the kind image you want.
+
+### Installation
+
+If you have go installed, you can run `go install sigs.k8s.io/kind@latest`.
```

Comment:
We need to mention that the kind version is linked to supported Kubernetes versions.
i.e. for K8s 1.25 you must have at least kind v0.15

Reply:
I actually don't think that is true.  I usually take the docker image as the version of Kubernetes.  

I think version v0.15 can support a wide range of kubernetes versions based on the image you use to create a cluster ie `kind create cluster --image=kindest/node:v1.25.2`

I do think the default version can change the default though so you are also probably correct.

Reply:
We had to specifically update kind for our e2e tests as older versions would not work with K8s 1.25. Kind version limits what you can run.

See the 3rd point in the [release notes for kind v0.15](https://github.com/kubernetes-sigs/kind/releases/tag/v0.15.0):
```
* Support for Kubernetes 1.25, fix for handling rootless + 1.25
```
And here is one from the [kind v0.13 release notes](https://github.com/kubernetes-sigs/kind/releases/tag/v0.13.0):
```
NOTE: You must use kind v0.13.0+ with Kubernetes v1.24.0+ images, and if you built your own Kubernetes v1.24.0+ image with a previous kind version you will need to re-built when switching to kind v0.13.0+.
```

Reply:
Ah.  I didn't know that.  Thanks for cleaning that up.  I will add this.

Reply:
Added.